docs/docs/general/04-Continuous Integration/askui-runner-Integration.md

```diff
@@ -0,0 +1,88 @@
+# AskUI CI/CD Integration
+
+You have two options to implement your pipeline. With our AskUI-Runner container or without it.
+
+The main difference without the AskUI-Runner is, that you can not download the latest version of your workflows from AskUI Studio inside your pipeline but have to provide them to your build.
+
+If you are more into reading just code we got you covered with a repository of all the pipeline code:
+
+[All the AskUI pipeline-files in a repository](https://github.com/askui/askui-ci)
+
+## AskUI-Runner: Get Workflows from AskUI Studio
+When you want to get the latest version of your workflows from AskUI Studio, the AskUI-Runner needs a configuration file `config.yaml` to know where to pull them from.
+
+The AskUI-Runner downloads the workflows and runs them inside a docker container. You do not have to take care of any installation of tools on your side 🥳.
+
+1. Set these environment variables for your pipeline:
+
+```shell
+ASKUI_WORKSPACE_ID=<Your workspace id from AskUI Studio>
+ASKUI_TOKEN=<An access token from your AskUI Studio workspace>
+ASKUI_INFERENCE_SERVER_URL=<Use https://inference.askui.com if your are not on-premise>
+ASKUI_WORKFLOW_ENDPOINT_URL=https://app-gateway-api.askui.com/prod/api/v1
+
+DOCKER_USERNAME=<Your Docker Hub username>
+DOCKER_PASSWORD=<Your Docker Hub password>
+```
+
+2. Create a `config.yaml` inside your pipeline like this:
+
+```bash
+# Example uses Github Actions syntax
+# You might have to change how to access 
+# environment variables!
+cat << EOF > config.yaml
+    access_token: ${{ secrets.ASKUI_TOKEN }}
+    inference_server_url: ${{ secrets.ASKUI_INFERENCE_SERVER_URL }}
+    workspace_id: ${{ secrets.ASKUI_WORKSPACE_ID }}
+    workflow_endpoint:
+        prefixes:
+            - workspaces/${{ secrets.ASKUI_WORKSPACE_ID }}/test-cases
+        url: ${{ secrets.ASKUI_WORKFLOW_ENDPOINT_URL }}/workspaces/${{ secrets.ASKUI_WORKSPACE_ID }}/objects            
+EOF
+```
+
+3. Execute your workflows with our AskUI-Runner container. Do not forget to login into Docker Hub 😉.
```

Comment:
We have not yet made the image public or did we? :D

Reply:
Oh yes. You are right! I totally forgot about it because with the credentials it worked 😉 

How should we go about this? Wait for the image, which should be released soon as far as I understood.

Reply:
Can we already merge the reporting stuff and move the ci stuff to a new pr?

Reply:
Done!